src/Microsoft.PowerShell.Commands.Management/cimSupport/cmdletization/cim/CimJobException.cs

```diff
@@ -50,6 +50,7 @@ public CimJobException(string message, Exception inner) : base(message, inner)
         /// </summary>
         /// <param name="info">The <see cref="SerializationInfo"/> that holds the serialized object data about the exception being thrown.</param>
         /// <param name="context">The <see cref="StreamingContext"/> that contains contextual information about the source or destination.</param>
+        [ObsoleteAttribute("Formatter-based serialization is obsolete and should not be used.")]
         protected CimJobException(
             SerializationInfo info,
             StreamingContext context) : base(info, context)
```

Comment:
Weird, the latest code in repo shows the obsolete attribute was already added to this method, and the `GetObjectData` method was already removed:
https://github.com/PowerShell/PowerShell/blob/master/src/Microsoft.PowerShell.Commands.Management/cimSupport/cmdletization/cim/CimJobException.cs#L53

Reply:
The change was done in https://github.com/PowerShell/PowerShell/pull/19696/files#diff-d60fa90db325d64ea4d5151107af7eeae9685143807a07c5e75f9c2568d47ad7R52 (#19696).
Did you sync with the upstream?